snap/info.go

```diff
@@ -538,6 +559,8 @@ func ReadInfo(name string, si *SideInfo) (*Info, error) {
 		return nil, err
 	}
 
+	info.renameClashingCorePlugs()
```

Comment:
Yeah, my concern here is just that we have two places currently where we fixup things snap.go. I would really like to have only a single place.